sdk/sdktypes/value_scalar.go

```diff
@@ -151,10 +151,23 @@ func (BooleanValue) isConcreteValue() {}
 
 func (s BooleanValue) Value() bool { return s.read().V }
 
-func NewBooleanValue(v bool) Value {
+func mkbool(v bool) Value {
 	return forceFromProto[Value](&ValuePB{Boolean: &BooleanValuePB{V: v}})
 }
 
+var (
+	True  = mkbool(true)
+	False = mkbool(false)
```

Comment:
Don't let readers get confused by `true` vs. `True`.
```suggestion
	TrueValue  = mkbool(true)
	FalseValue = mkbool(false)
```

Reply:
done, begrudgingly